src/preview-app.html

```diff
@@ -146,6 +153,7 @@
             </paper-tab>
           </paper-tabs>
         </template>
+        <paper-button raised on-tap="_signout" hidden$="[[_showSignOut(page.page)]]">Sign out</paper-button>
```

Comment:
`_showSignOut` is not needed. If the `paper-button` is moved inside the `template`, it will hide when the current page is the home page.